src/context/GameContext.js

```diff
@@ -0,0 +1,42 @@
+import { useContext } from 'react';
+import { createContext, useState } from 'react';
+import initialCards from '../cards-data';
+
+const GameContext = createContext();
+const GameProvider = ({ children }) => {
+  const [deck, setDeck] = useState(initialCards);
+  const [playerOneHand, setPlayerOneHand] = useState([]);
+  const [selectedCard, setSelectedCard] = useState();
+  const [playerTwoHand, setPlayerTwoHand] = useState([]);
+  const [playerThreeHand, setPlayerThreeHand] = useState([]);
+  const [from, setFrom] = useState('deck');
+  const [to, setTo] = useState(1);
+
+  return (
+    <GameContext.Provider
+      value={{
+        deck,
+        setDeck,
+        playerOneHand,
+        setPlayerOneHand,
+        selectedCard,
+        setSelectedCard,
+        playerTwoHand,
+        setPlayerTwoHand,
+        playerThreeHand,
+        setPlayerThreeHand,
+        from,
+        setFrom,
+        to,
+        setTo,
+      }}
+    >
+      {children}
+    </GameContext.Provider>
+  );
+};
+const useGameContext = () => {
+  const context = useContext(GameContext);
+  return context;
+};
```

Comment:
the only purpose of this function is to add a check that will add an error message if someone goes wrong using the context but since you're not doing that, you'd be better off exporting ```GameContext``` then when you call it in the component you want to access context ```const { stuff you want from context } = useContext(GameContext)```